lib/panoptes-subject-uploader.iced

```diff
@@ -118,42 +141,84 @@ for file in args._
     log "On row #{i + 1} of #{rows.length}"
 
     metadata = getMetadata row
-    imageFileNames = findImages path.dirname(file), metadata
+    
+    if args.skipMediaUpload
+
+      subject = {}
+      subject.metadata = metadata
+      subject.locations = []
+      subject.links = project: args.project
+
+      # find a https urls for the row
+      imageURLs = findImagesURLs metadata
+      if imageURLs.length is 0
+        log "!!! Cannot find a https url for row #{i + 1}"
+        break
+      
+      for url, url in imageURLs
```

Comment:
Maybe I'm missing here, but this is confusing to me: both the value and index are using `url`